usaspending_api/download/filestreaming/__init__.py

```diff
@@ -0,0 +1 @@
+NAMING_CONFLICT_SUFFIX = "_NAMING_CONFLICT_SUFFIX"
```

Comment:
Nice trick. I agree with your comment about _simpler is better than complex_ here.

Only because I think too much about names, maybe "DISCRIMINATOR" is a better term than "SUFFIX" to have in this thing? https://www.google.com/search?q=define%3Adiscriminator